app/uk/gov/hmrc/cgtpropertydisposals/connectors/SubscriptionConnector.scala

```diff
@@ -81,6 +69,22 @@ class SubscriptionConnectorImpl @Inject()(http: HttpClient, val config: Services
     )
   }
 
+  override def getSubscription(cgtReference: CgtReference)(
+    implicit hc: HeaderCarrier
+  ): EitherT[Future, Error, HttpResponse] = {
+
+    val queryParameters = Map("regime" -> "CGT", "id" -> cgtReference.value)
+
+    EitherT[Future, Error, HttpResponse](
+      http
+        .get(baseUrl + "/subscriptions", queryParameters, headers)
```

Comment:
`(baseUrl + "/subscriptions"` is essentially a static value - it could be factored outside of this method to avoid the concatenation and assignment to a temporary value every time this method is called

Reply:
fixed